firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -384,12 +384,16 @@ off_below: 0.10
 pin: PB5
 fan_speed: 0.3
 kick_start_time: 0.5
+heater: heater_bed, extruder
+stepper: stepper_x, stepper_y, stepper_y

```

Comment:
I would remove the stepper line, because the default is *any* stepper, which is what we want. less confusing that way. 

Also remove the space between the items in 387.

Reply:
There you go